README.md

```diff
@@ -24,6 +24,61 @@ cd $(The repo you just cloned)
 
 ## 2. Install all gems
 
+### Installation for Windows Users
+
+1. Install Ruby
+If you are on windows you will need to install `ruby`
+
+https://rubyinstaller.org/downloads/
+We suggest installing ruby version `2.6.6.1+dev kit`. Other versions
+may work but have not been tested.
+
+When you install ruby, be sure that you also check the box to install `MSYS2`.
+
+at the end of the `ruby` install, it will ask you if you want to install `MSYS2`.
+Install the base installation
+
+2. Next, install the bundler.
+
+`gem install bundler`
```

Comment:
Maybe clarify that this line needs to be run in Git Bash or a similar program? Windows users don't normally run commands like that and might get confused just seeing a line of code in the install instructions.